src/style-property-map-readonly.js

```diff
@@ -103,27 +92,22 @@
 
   };
 
-  ComputedStylePropertyMap.prototype.set = function(property, value) {
-    throw new TypeError('ComputedStylePropertyMap is immutable');
-  };
-
-  ComputedStylePropertyMap.prototype.getProperties = function() {
-    var computedStyles = window.getComputedStyle(this._element);
+  StylePropertyMapReadOnly.prototype.getProperties = function() {
     var output = [];
-    for (var i = 0, l = computedStyles.length; i < l; ++i) {
-      var property = computedStyles[i];
+    for (var i = 0, l = this._styleObject.length; i < l; ++i) {
+      var property = this._styleObject[i];
       // TODO: Construct the objects for each type of thing and add them to the
       // output array.
     }
     return output;
   };
 
-  function getComputedStyleMap(element) {
-    return new ComputedStylePropertyMap(element);
-  }
+  scope.getComputedStyleMap = function(element) {
```

Comment:
Please make this 
```
getComputedStyleMap = function(element)
```
and put
```
scope.getComputedStyleMap = getComputedStyleMap
```
below (with the other scope member assignments).